sql/CMakeLists.txt

```diff
@@ -68,6 +68,8 @@ set(TS_MODULE_PATHNAME
     ${MODULE_PATHNAME}
     PARENT_SCOPE)
 
+set(TSL_MODULE_PATHNAME "$libdir/timescaledb-tsl-${PROJECT_VERSION_MOD}" PARENT_SCOPE)
```

Comment:
That seems unrelated change.

Reply:
It is used in the isolation test, we load different module there

Reply:
this is non-TSL parts of the code so you shouldnt be referencing tsl-stuff here

Reply:
Not sure why, this is a tsl-part isolation test. We use this approach everywhere in tsl tests to link C functions from `tsl/test/src`